### ISSUE TITLE
package for components on your board?

### ISSUE THREAD

**Issue by batchku** [mo-seph/PiezoPreamp#4]
hello,
i'd like to try this board out.

i've found the SS201 JFETs:
http://www.mouser.com/Search/Refine.aspx?Keyword=sst201

but it seems to be "obsolete".

do you have a replacement in mind?

also, have you come across these monolithic dual-channel JFETs in the SOIC-8 and TO-78 packages:
http://www.vishay.com/company/press/releases/2003/030502jfet/

they may be better suited to this application?

thank you in advance!

ali momeni

**Comment by mo-seph** [comment]
Hi,

Glad you're interested! I've got a farnell parts list, ill try to send it in the next couple of days. I think everything is 1208 or 1206.

I'd be up for swapping ideas about construction if you want...

Cheers,

## dave 

Sent from my Android phone with K-9 Mail. Please excuse my brevity.

batchku reply@reply.github.com wrote:

hello,
i'd like to try this board out.
would it be possible for you to provide me with links to the packages you're using for the components?
i've found the SS201 JFETs, but the resistors and capacitors.... what size are you? digikey links?

thank you in advance!

ali momeni

---

Reply to this email directly or view it on GitHub:
https://github.com/mo-seph/PiezoPreamp/issues/4
The University of Edinburgh is a charitable body, registered in
Scotland, with registration number SC005336.

**Comment by batchku** [comment]
hi there,
thanks a lot for the response;
i'll be happy to talk about the board with you.

and i found the packages from the eagle files; all are 1206 it seems...

planning on etching one this week!

ali

**Comment by mo-seph** [comment]
Here's a csv file from Farnell of the parts.

Let me know how it goes! I just spent last night doing some JFET matching, going to build up a few more of these tonight...

Cheers,
dave

**Comment by batchku** [comment]
Have u looked into dual channel fet ICs?

**Comment by mo-seph** [comment]
Yup, had a look, couldn't find anything sensible. IIRC there's some analogue devices parts, but they're about £5 each, and hard to get hold of. If you find anything, I'd be interested!

I the end, I figured this way was probably more accessible - I want to get a small run of boards done so that people can build their own preamps, and it seems easier to get hold of the single jfets. But then again, matching is a pain! I might try not matching a few to see how much difference it makes.

Cheers,
dave

**Comment by batchku** [comment]
also, regarding contact/piezo mic amplification,
did you look into a "charge amp" instead of a FET?

charge amp:
http://en.wikipedia.org/wiki/Charge_amplifier

great reference:
http://www.ti.com/lit/an/sloa033a/sloa033a.pdf

more....
http://www.diyaudio.com/forums/instruments-amps/203183-charge-amplifier-guitar-preamp-questions.html
http://e2e.ti.com/support/amplifiers/audio_amplifiers/f/6/t/136962.aspx
http://www.mmf.de/instrumentation.htm
http://www.dytran.com/go.cfm/en-us/content/tech-education-a1/x
http://sales.hamamatsu.com/assets/applications/SSD/charge_amp_kacc9001e01.pdf
http://www.engineerlive.com/Design-Engineer/Instrumentation/Multichannel_charge_amplifier_suits_high-accuracy_measurements/22893/

**Comment by mo-seph** [comment]
Yeah, tried a few different things. The thing that drew me to this design in the end was that phantom power was part of it. I couldn't get anything opamp based to work off phantom. It might also just be my field recorder, but opamps seem to need around 10mA, any my kit will only do about 3-4. Not sure how much the JFET uses, but it's really not very much.

I'd love to do a comparison at some point, because there are a whole bunch of different possibilities out there, and it's not easy to choose between them.

A paper that got me really excited is attached...

Cheers,
dave

**Comment by batchku** [comment]
hi dave,
yes, i would love a comparison also;
the paper that got you excited didn't come through; i'd like to take a look.
would you please direct me to it/send it over?  batchku@gmail.com is me

ali

**Comment by batchku** [comment]
hello dave,
are you still continuing this development?
did you ever get a run of them made?
i'm also keen on getting a few made as we could really use them in this class:
http://teach.alimomeni.net/2012fall1/

also, the cvs file of the farnel component list also didn't come thru (like the paper you liked).
would you please hit me with those again?

thank you,
ali momeni
batchku@gmail.com
